fem/intrules.cpp

```diff
@@ -60,6 +61,7 @@ IntegrationRule::IntegrationRule(IntegrationRule &irx, IntegrationRule &iry,
    const int nz = irz.GetNPoints();
    SetSize(nx*ny*nz);
    SetPointIndices();
+   Order = std::min(std::min(irx.GetOrder(), iry.GetOrder()), irz.GetOrder());
```

Comment:
A little bit clearer:

```suggestion
   Order = std::min({irx.GetOrder(), iry.GetOrder(), irz.GetOrder()});
```